.gitignore

```diff
@@ -101,6 +101,7 @@ gradle-app.setting
 .idea/sqlDataSources.xml
 .idea/dynamic.xml
 .idea/uiDesigner.xml
+/.idea/
```

Comment:
We decided to keep some of the idea files, by using the "intellij+iml" configuration provided by gitignore.io. Remove this line please.

Reply:
done